metadata.json

```diff
@@ -15,6 +15,14 @@
     {
       "name": "puppetlabs/stdlib",
       "version_requirement": ">= 4.13.0 < 6.0.0"
+    },
+    {
+      "name": "puppetlabs/apt",
+      "version_requirement": ">= 6.3.0 < 7.0.0"
```

Comment:
the newest release is 7.0.1. Is there a reason why you don't allow 7.x?

Reply:
Well, right. Minor updates should be ok. :)